Main.java

```diff
@@ -9,7 +10,26 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // declare a variable that will calcualte the quotient of two numbers and the remainder
+    String name;
+   System.out.println("Enter two numbers using X and Y to calcualte the quotient and the remainder:");
+    
+    // Creates a Scanner used for input
+    Scanner input = new Scanner(System.in);
+     int X = input.nextInt();
+
+     // Creates a Scanner used for input
+     int Y = input.nextInt();
+
+
+     int Z = (X/Y);
+     int P = (X%Y);
```

Comment:
Remember that variable names should always begin with a lowercase letter and describe what they are storing or being used for. While x and y would be ok for the numbers, the name number1 and number2 might be  a better choice. Similarly, Z and P should really be quotient and remainder, as that is what they are. Also, your comment on line 21 isn't making a Scanner. It is using the scanner to get user input.